fml/status.h

```diff
@@ -0,0 +1,91 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FML_STATUS_H_
+#define FLUTTER_FML_STATUS_H_
+
+#include "flutter/fml/string_view.h"
+
+namespace fml {
+
+/// Google's canonical list of errors, should be in sync with:
+/// https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto
+enum class StatusCode : int {
+  kOk = 0,
+  kCancelled = 1,
+  kUnknown = 2,
+  kInvalidArgument = 3,
+  kDeadlineExceeded = 4,
+  kNotFound = 5,
+  kAlreadyExists = 6,
+  kPermissionDenied = 7,
+  kResourceExhausted = 8,
+  kFailedPrecondition = 9,
+  kAborted = 10,
+  kOutOfRange = 11,
+  kUnimplemented = 12,
+  kInternal = 13,
+  kUnavailable = 14,
+  kDataLoss = 15,
+  kUnauthenticated = 16,
+  kDoNotUseReservedForFutureExpansionUseDefaultInSwitchInstead_ = 20
+};
+
+/// Class that represents the resolution of the execution of a procedure.  This
+/// is used similarly to how exceptions might be used, typically as the return
+/// value to a synchronous procedure or an argument to an asynchronous callback.
+class Status final {
+ public:
+  /// Creates an 'ok' status.
+  Status();
+
+  Status(fml::StatusCode code, fml::StringView message);
+
+  fml::StatusCode code() const;
```

Comment:
Here and elsewhere, fml:: is redundant.

Reply:
It's not redundant in all cases.  It can be ambiguous so that's why it is explicit here and in absl.